wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,27 @@
+<form [formGroup]="registerForm" (ngSubmit)="addPost(registerForm.value)" class="form-container" novalidate>
+  <header class="header">
+    <img src="assets/wolox_logo.svg" class="header-logo" alt="Wolox logo">
+    <h1 class="header-title">BOOKS</h1>
+  </header>
+  <div class="form-signup">
+    <label class="form-field">First name</label>
+    <input type="text" formControlName="firstName" class="form-input">
+    <p class="alert" *ngIf="!registerForm.controls['firstName'].valid && registerForm.controls['firstName'].touched">{{ fieldRequiredAlert }}</p>
+    <label class="form-field">Last name</label>
+    <input type="text" formControlName="lastName" class="form-input">
+    <p class="alert" *ngIf="!registerForm.controls['lastName'].valid && registerForm.controls['lastName'].touched">{{ fieldRequiredAlert }}</p>
+    <label class="form-field">Email</label>
+    <input type="text" formControlName="email" class="form-input">
+    <p class="alert" *ngIf="!registerForm.controls['email'].valid && registerForm.controls['email'].touched">Please enter a valid email.</p>
+    <label class="form-field">Password</label>
+    <input type="password" formControlName="password" class="form-input">
+    <p class="alert" *ngIf="!registerForm.controls['password'].valid && registerForm.controls['password'].touched">Password must contain at least 8 characters.</p>
+    <label class="form-field" *ngIf="registerForm.controls['password'].valid">Confirm Password</label>
+    <input type="password" formControlName="confirmPassword" class="form-input" *ngIf="registerForm.controls['password'].valid">
+    <p class="alert" *ngIf="registerForm.controls.confirmPassword.errors?.MatchPassword && registerForm.controls['confirmPassword'].touched && registerForm.controls['password'].valid">Password does not match the confirm password.</p>
+  </div>
+  <ul class="buttons">
+    <li class="buttons-elem"><button type="submit" class="button" [disabled]="!registerForm.valid">Sign up</button></li>
```

Comment:
There's no need for the list, you can do something like:

```html
<div class="buttons">
  <button type="submit" class="button" [disabled]="...">Sign up</button>
  <button type="button" class="button">Login</button>
</div>
```

Though I really think you should use another class names for buttons, something a little less generic